### ISSUE TITLE
classification layers arent training

### ISSUE THREAD

**Issue by kotvkaske** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#86]
Good morning! I am using SSD architecture to detect faces on images (2 classes - background and face). I have a labeled WIDERFACES dataset and Im trying to train a model on it. However on each epoch after using softmax each box has 50-50 confidence which means classification layer is not training .What should I do?
